deploy/chart/devfile-registry/templates/deployment.yaml

```diff
@@ -0,0 +1,109 @@
+#
+# Copyright (c) 2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ template "devfileregistry.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+  name: {{ template "devfileregistry.fullname" . }}
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: {{ template "devfileregistry.name" . }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
```

Comment:
the `maxUnavailable` sounds a bit high for an update criteria

Reply:
TBH, yeah I agree with you. It's also not really necessary since our chart only manages one pod (versus a multi-pod chart where this field might be more useful). 

Removing this field really shouldn't impact our update strategy, so I'll take it out.